web/src/app/dialogs/components/DialogTitleWrapper.js

```diff
@@ -36,10 +37,11 @@ const styles = theme => {
  * Renders a fullscreen dialog with an app bar if on a small
  * or mobile screen, and a standard dialog title otherwise.
  */
+@isFullScreen()
 @withStyles(styles)
 export default class DialogTitleWrapper extends Component {
   static propTypes = {
-    fullScreen: p.bool.isRequired,
+    fullScreen: p.bool,
```

Comment:
Is this related to drag-and-drop or fixing another issue?

Reply:
Yeah, in my Additional Info section, I added 
> Removes requirement for `fullScreen` prop passed to `DialogTitleWrapper`
to make implementing something easier in relation to fixing the DND issue